gra.md

```diff
@@ -26,3 +28,48 @@ pojedynczo tutaj).
 Gra jest podatna na wypaczenie ze względu na rozumienie jej zasad, w
```

Comment:
Zastanowić się, gdzie jest jeszcze jedna cecha, jaka jest relacja gry do sytuacji, w której extrinsic factors zaczynają wypaczać sens podejmowania działań. Czy to powinno być częścią definicji, czy nie?